python.html.markdown

```diff
@@ -144,6 +145,10 @@ bool("")  # => False
 # Python has a print statement
 print "I'm Python. Nice to meet you!"
 
+# Simple way to get input data from console
+input_string_var = raw_input("Enter some data: ") # Data is stored as a string
+input_number_var = input("Enter a number: ") # Data is stored as a number
```

Comment:
@levibostian , Hi, my mistake, thank you for correcting. input() treats the input data as python code and tries to execute it. Yes, It evaluates numbers as well as python code given as input data.
Normally, it's use is not recommended and it's been deprecated in Python 3. Should input() method be actually included in this documentation? What do you think? Please let me know, I will commit the changes. Thanks.